experimental/Schemes/Auxiliary.jl

```diff
@@ -126,6 +126,16 @@ function pullback(f::AbsCoveredSchemeMorphism, C::EffectiveCartierDivisor)
   return EffectiveCartierDivisor(X, triv_dict, trivializing_covering=triv_cov, check=false)
 end
 
+function pullback(f::AbsCoveredSchemeMorphism, C::CartierDivisor)
```

Comment:
Had this only been done for effective cartier divisors?

Reply:
I think so.